cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,108 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  const worker = {
+    firstName: 'Alex',
+    lastName: 'User',
+    email: 'alexuser@gmail.com',
+    age: 24,
+    salary: 24000,
+    department: 'PR'
+  };
 
+  const changedWorker = {
+    firstName: 'Sasha',
+    lastName: 'User',
+    email: 'sashauser@gmail.com',
+    age: 23,
+    salary: 12000,
+    department: 'QA'
+  };
+
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should have pagination', () => {
+    cy.get('.-pagination').should('contain', 'Previous');
+    cy.get('.-pagination').and('contain', 'Next');
+    cy.contains('.-pageInfo', 'Page').should('exist');
+  });
+
+  it('should have rows count selection', () => {
+    cy.get('select').select('5 rows');
+    cy.get('select').should('contain', '5 rows');
+  });
+
+  it('should add new worker', () => {
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(worker.firstName);
+    cy.get('#lastName').type(worker.lastName);
+    cy.get('#userEmail').type(worker.email);
+    cy.get('#age').type(worker.age);
+    cy.get('#salary').type(worker.salary);
+    cy.get('#department').type(worker.department);
+    cy.get('#submit').click();
+
+    cy.get('.rt-tbody').should('contain', worker.firstName);
+    cy.get('.rt-tbody').and('contain', worker.lastName);
+    cy.get('.rt-tbody').and('contain', worker.email);
+    cy.get('.rt-tbody').and('contain', worker.age);
+    cy.get('.rt-tbody').and('contain', worker.salary);
+    cy.get('.rt-tbody').and('contain', worker.department);
+  });
+
+  it('should delete a worker', () => {
+    cy.get('#delete-record-1').click();
+    cy.get('.rt-tbody').should('not.contain', '#delete-record-1');
+  });
+
+  it('should delete all workers', () => {
```

Comment:
The test for deleting all workers is not scalable. If you have more than 3 workers, they will not be deleted. Consider implementing a loop that will continue to delete workers until none are left.